src/ConsoleContactAppManager/Contact.cs

```diff
@@ -0,0 +1,79 @@
+﻿// <copyright file="Contact.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace ContactManagerApp
+{
+    /// <summary>
+    /// Class Contact has  name, phone number, email, notes.
+    /// </summary>
+    public class Contact
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Contact"/> class.
+        /// Constructor Contact passes the input from user.
+        /// </summary>
+        /// <param name="name">Name as string</param>
+        /// <param name="phoneNumber">PhoneNumber</param>
+        /// <param name="email">Email of object 'contact'</param>
+        /// <param name="notes">Has additional notes</param>
+        /// <param name="id">Has the unique id for the contact</param>
+        public Contact()
+        {
+            this.Id = 0;
```

Comment:
Fill default values in the properties itself

Reply:
Default values are given in the properties.